package.json

```diff
@@ -40,7 +40,7 @@
     "lodash": "^3.3.0",
     "mocha-mongoose": "^1.0.3",
     "mongodb": "^1.4.33",
-    "mongoose": "^3.8.23",
+    "mongoose": "3.8.23",
```

Comment:
sry really wanted to change this...the unstable version of mongoose thing is killing me